pkg/security/probe/activity_dump.go

```diff
@@ -201,11 +247,20 @@ func (ad *ActivityDump) dump() {
 		seclog.Errorf("couldn't marshal ActivityDump: %s\n", err)
 		return
 	}
-	_, err = ad.outputFile.Write(raw)
+	n, err := ad.outputFile.Write(raw)
 	if err != nil {
 		seclog.Errorf("couldn't write ActivityDump: %s\n", err)
 		return
 	}
+
+	// send dump size
+	if n > 0 {
+		var tags []string
+		if err = ad.adm.probe.statsdClient.Gauge(metrics.MetricActivityDumpSizeInBytes, float64(n), tags, 1.0); err != nil {
+			seclog.Warnf("couldn't send %s metric: %w", metrics.MetricActivityDumpSizeInBytes, err)
```

Comment:
```suggestion
			seclog.Warnf("couldn't send %s metric: %v", metrics.MetricActivityDumpSizeInBytes, err)
```
Warnf doesn't support %w

Reply:
👍🏻